tdms/tests/matlab/test_iteratefdtd_matrix.m

```diff
@@ -0,0 +1,55 @@
+
+function tests = test_iteratefdtd_matrix_function
+tests = functiontests(localfunctions);
+end
+
+function testInvalidIlluminationSourceIJKErrors(testCase)
+    verifyError(testCase, @()createInputWithInvalidIlluminationSourceIJ(), 'TDMSException:InvalidIlluminationFile');
+    delete *.mat
```

Comment:
Probably better to create and then delete a specific directory?

Reply:
Ah yes – the right way. Done now 🎉